src/reducers/entities/channels.js

```diff
@@ -437,6 +448,9 @@ export default combineReducers({
     // object where every key is the channel id and has and object with the channel detail
     channels,
 
+    // object where every key is the channel id and has and object with the channel detail for all channels list
+    allChannels,
```

Comment:
Why the new reducer? Why not just use the `channels` reducer?

Reply:
@jespino ☝️

Reply:
The same reason here, is to have a different place to list or iterate over a bunch of all the channels, not the "channels loaded to use allover the application"